docs/root/configuration/well_known_dynamic_metadata.rst

```diff
@@ -0,0 +1,20 @@
+.. _well_known_dynamic_metadata:
+
+Well Known Dynamic Metadata
+===========================
+
+Filters can emit dynamic metadata via the `StreamInfo` interface on a `Connection`. Following are
+some well known dynamic metadata keys emitted by Envoy filters.
+
+RBAC Filter
```

Comment:
I think the dynamic metadata should live direclty in the filter docs. This general section can just talk about the concept?

Reply:
Addressed in eaf0e89.